ui/app/css/design-system/colors.scss

```diff
@@ -74,3 +74,66 @@ $accent-pink: #ff45d8;
 $neutral-white: #fff;
 $neutral-black: $Black-100;
 $neutral-grey: $Grey-500;
+
+// Everything below this line is part of the new color system
+$primary-1: #037dd6;
+$primary-2: #eaf6ff;
+$primary-3: #0260a4;
+
+$secondary-1: #f66a0a;
+$secondary-2: #fef5ef;
+$secondary-3: #c65507;
+
+$error-1: #d73a49;
+$error-2: #fcf2f3;
+$error-3: #b92534;
+
+$alert-1: #ffd33d;
+$alert-2: #fefcde;
+$alert-3: #f8c000;
+
+$success-1: #4cd964;
+$success-2: #caf4d1;
+$success-3: #219e37;
+
+$ui-black: #24292e;
+$ui-white: #fff;
+$ui-1: #f2f3f4;
+$ui-2: #d6d9dc;
+$ui-3: #bbc0c5;
+$ui-4: #6a737d;
+
+$mainnet: #29b6af;
```

Comment:
Where were these defined? I don't see these with the other colours in Figma.

Reply:
Oh, actually I found all of the others hard-coded in our network menu. All except Mainnet, which has a different colour at the moment.

Reply:
I proposed them to be added [here](https://consensys.slack.com/archives/G017VDA9EKS/p1610659081000400), but no response. I lifted these colors from elsewhere in the codebase but did not replace the usage in this PR because I later refactored those components.

Reply:
I was wrong about Mainnet - that one is the same as well. My search failed to find it because it was all-caps.